DirectEmployers/MyJobs
# [PD-3737] history not working multi selects

## What's this PR do?
PR fixes the history back and forward buttons for the multi-select combos

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside of the analytics application

## How should this be manually tested?
Pull down repository. Go into site traffic reports and select from the multi-select combos. Use the back and forward buttons to make sure history is working

## Any background context you want to provide?

## PR Checklist

### Front-End


### Separation of Concerns


### Tests


### Migrations
